### PR TITLE
[EXAMPLE] Popover - static

### PR DESCRIPTION
**Pull request by aibcmars**
Add custom popover for the task found by it's id

https://cdn.statically.io/gh/process-analytics/bpmn-visualization-examples/execution-data_popover_info/examples/index.html

### CODE REVIEW

File: examples/custom-navigation/popover/index.js [hunk 1]
```diff
@@ -0,0 +1,196 @@
+// TODO duplicated with the 'diagram-fit-variants' example
+function getBpmnDiagramHorizontal() {
```
Comment (tbouffard) [inline examples/custom-navigation/popover/index.js:2]: :warning: should use #111

**Comment by csouchet** [comment]
## MacOs
### Firefox
_**No Pop Over 🙁**_ 
image

### Chrome
image

### Safari
_**No Pop Over 🙁**_ 
image

File: examples/custom-navigation/popover/index.js [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+  popoverContainer.setAttribute('style', 'position: absolute;');
+  const boundingRect = popoverContainer.getBoundingClientRect();
+  const popoverHeight = boundingRect.height;
+  // speech bubble indicator height is 21
+  const style = `position: absolute; top: -${popoverHeight + 21}px`;
+  popoverContainer.setAttribute('style', style);
+  popoverContainer.classList.remove('hidden');
+}
+
+function setButtonState(btnId, disabled) {
```
Comment (csouchet) [inline examples/custom-navigation/popover/index.js:184]: ❓ Can we put the method for the button behaviour in another script, and keep here only the code related to the popover configuration ?

**Comment by csouchet** [comment]
I tought a popover is displayed only when the mouse is over the HTML element with the listener, like: https://picturepan2.github.io/spectre/components/popovers.html

File: examples/custom-navigation/popover/index.js (hunk 1 above)
Comment (aibcmars) [reply examples/custom-navigation/popover/index.js:2]: waiting for merge of #111

File: examples/custom-navigation/popover/README.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+After the main `BpmnVisualization` instantiation, get the HTML element corresponding to the id from search input.
+
+⚠️  In order to avoid having to many content in the README, we simplify it. You can find all the content of the example in [index.js](index.js).
+What is important to know:
+- Definition of style for the popover are in index.html
+- The HTML passed as the content of the popover is not sanitized - it is up to integrator to do so.
+
+
+⚠️ It's not possible to use navigation of the diagram at the moment. \
```
[the remark this answers is not in the document]
Comment (aibcmars) [reply examples/custom-navigation/popover/README.md:17]: sure

File: examples/custom-navigation/popover/README.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
+# Add a popover with information to the task found by it's id
```
[the remark this answers is not in the document]
Comment (aibcmars) [reply examples/custom-navigation/popover/README.md:1]: ok

File: examples/custom-navigation/popover/index.html [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+                overflow: hidden;
+            }
+            #controls {
+                margin-top: 1em;
+            }
+            .mw-200 {
+                min-width: 200px;
+            }
+
+            .chartreuse rect {
```
[the remark this answers is not in the document]
Comment (aibcmars) [reply examples/custom-navigation/popover/index.html]: Yes, actually you are right,
Adding this differentiation can add unnecessary complexity.

I am OK to remove it - let me know if you want to discuss anything in particular or I can proceed with removal

**Comment by aibcmars** [comment]
@csouchet 
I have changed the sequence of disappearing / appearing elements for popover
it helped for Firefox display on Windows.

please confirm it is OK now for Mac:
- [ ] Firefox
- [ ] Safari

**Comment by tbouffard** [comment]
:heavy_check_mark: display ok on Ubuntu 18.04, FF 83.0 and Chromium 87.0.4280.88 for commit 131456c

**Comment by csouchet** [comment]
The div appears well in Firefox & Chrome, not Safari, but it appears all time even if I don't move my mouse on the task. 
For me, I don't consider it as a popover.

File: examples/custom-navigation/popover/index.js (hunk 1 above)
Comment (tbouffard) [reply examples/custom-navigation/popover/index.js]: rebased on master after #11 merge

**Review by tbouffard — APPROVED** [review]

**Pull request merged**